platform/osx/osx.xcodeproj/project.pbxproj

```diff
@@ -1018,6 +1022,10 @@
 				HEADER_SEARCH_PATHS = ../../include;
 				INFOPLIST_FILE = test/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/../Frameworks";
+				OTHER_CPLUSPLUSFLAGS = (
+					"$(OTHER_CFLAGS)",
+					"$(geometry_cflags)",
+				);
```

Comment:
@jfirebaugh how come we need this here in addition to https://github.com/mapbox/mapbox-gl-native/blob/geometry.hpp/platform/osx/osx.xcodeproj/project.pbxproj#L985?